src/pages/SideBar/SideBar.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { Menu } from 'semantic-ui-react';
+import styled from 'styled-components';
+const SideMenu = styled(Menu)({
+  height: '100%',
+});
+
+const Item = styled(Menu.Item)((props) => ({
+  background: props.active
+    ? `-webkit-linear-gradient(120deg,
+  #833ab4 0,
+  #fd1d1d 20%,
+  #e36414 80%) !important;
+  -webkit-background-clip: text !important;
+  -webkit-text-fill-color: transparent !important
+  `
+    : 'transparent',
+}));
+
+function SideBar(props) {
+  const { logout } = props;
+  const [optionMenu, setOptionMenu] = React.useState('home');
+
+  return (
+    <SideMenu fluid vertical tabular>
+      <Item
+        icon="home"
+        name="home"
+        active={optionMenu === 'home'}
+        onClick={() => setOptionMenu('home')}
+      />
+      <Item
+        icon="heart outline"
+        name="favorites"
+        active={optionMenu === 'favorites'}
+        onClick={() => setOptionMenu('favorites')}
+      />
+      <Item
+        icon="clock outline"
+        name="watchLater"
+        active={optionMenu === 'watchLater'}
+        onClick={() => setOptionMenu('watchLater')}
+      />
```

Comment:
this can be changed to a loop instead of having to manually define each item:

```jsx
const items = [
  { name: 'home', icon: 'home' },
  { name: 'favorites', icon: 'heart outline' },
  { name: 'watchLater', icon: 'clock outline' },
];


...

{items.map(({ name, icon }) => (
  <Item
    icon={icon}
    name={name}
    active={optionMenu === name} 
    onClick={() => setOptionMenu(name)}
  />
))}

...
```